libs/prelude/Prelude/Basics.idr

```diff
@@ -29,7 +29,7 @@ fst (x, y) = x
 snd : (a, b) -> b
 snd (x, y) = y
 
-infixl 9 .
+infixr 9 .
```

Comment:
Why the re-association?

Reply:
Three minor reasons, which may or may not add up to a good enough reason to change it.
* It now matches Haskell's associativity of this operator.
* It better represents the order in which functions will be applied (`f . (g . (h . i))` vs old `((f . g) . h) . i`).
* There is no difference in functionality (no pun intended).